test/test_tabbar_component_spec.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import assert from "assert";
+import {mount} from "enzyme";
+
+import {Tabbar} from "../src/components/tabbar/tabbar";
+
+describe("<Tabbar />", () => {
+    it("defaults to selecting the first item", () => {
+        // Arrange
+        const wrapper = mount(
+            <Tabbar
+                items={["Numbers", "Geometry", "Operators"]}
+                onSelect={() => {}}
+            />,
+        );
+
+        // Assert
+        expect(wrapper).toHaveState("selectedItem", 0);
+        const firstItem = wrapper.find("TabbarItem").first();
+        expect(firstItem).toHaveProp("itemState", "active");
```

Comment:
In general it's better to assert things about what's render as opposed to internal state.  This allows you to modify the implementation without having to update tests as much.  I didn't bring this up during our pairing sessions yesterday because I was focused on getting things working and showing off some of the different types of assertions that can be made.  There may be some times where checking the internal state makes sense.

I think it might be worth adding assertions to check that the other items are inactive.